FreshRSS/FreshRSS
# Reading view: action icons position

inspired by #6261 from @joshka

### (default): above title
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/e64eaa9a-54b0-43e0-a324-d712989a7ede)

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/c6936e4a-4694-4a63-bda1-34696dfbef25)



### in authors and date row
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/7bc128f7-f054-4a43-b6ec-eb0d06311678)

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/4055874d-b360-4161-96af-e991e98b03ea)

### top line article icons configs respected
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/2b351d27-c63c-4519-b58d-15917faddd8c)




How to test the feature manually:

1. go to config `reading`, section `Articles: header/footer`
2. edit the option
3. see in reading view

Pull request checklist:
